src/lv_interop.h

```diff
@@ -52,6 +52,7 @@ namespace grpc_labview
     typedef int32_t MagicCookie;
     typedef MagicCookie LVRefNum;
     typedef MagicCookie LVUserEventRef;
+    typedef int32_t(*CleanupProcPtr)(gRPCid* data);
```

Comment:
should we call it id here instead of data? is there some convention for declaring cleanuprocs?

Reply:
Updated.